releases/v0.8.0-pre0.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
may be we should call this v0.8.0-shipyard-pre0.yaml ? 

So we can then keep the other items in the repo?

Reply:
In a way that we would keep:

- v0.8.0-pre0-shipyard
- v0.8.0-pre0-admiral
- v0.8.0-pre0-submariner-lighthouse
- v0.8.0-pre0-release

?

Reply:
The idea is to continuously update the same file until its all released.

Reply:
I don't see a point in keeping separate files, its just different stages of the same release process. Once you've finished the previous stage you continue to the next one (and don't update the hash of the already released parts)

Reply:
Yeah, but for documentation purposes, I think it's helpful to keep, at least for now, so the process is fully traceable. We could do that via git logs too.

What you say also makes sense I guess :)